.github/workflows/release-containers.yml

```diff
@@ -60,113 +89,183 @@ jobs:
-        asset_content_type: application/zip
+  push-manifest:
+    runs-on: ubuntu-latest
+    needs: [set-version, build-and-push]
+    permissions:
+      contents: write
+      packages: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
 
-    - name: Zip Full Dist
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      run: tar -cvf python-package-distributions.tar.gz ./dist
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
 
-    - name: Upload Full Dist
-      uses: actions/upload-release-asset@v1
-      if: github.event_name != 'push' || startsWith(github.ref, 'refs/tags/')
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      with:
-        upload_url: ${{ steps.create_release.outputs.upload_url }}
-        asset_path: python-package-distributions.tar.gz
-        asset_name: python-package-distributions.tar.gz
-        asset_content_type: application/zip
+      - name: Login to ECR
+        uses: docker/login-action@v3
+        with:
+          registry: 471112909428.dkr.ecr.us-east-1.amazonaws.com
+          username: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          password: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
-    - name: Upload the distribution packages
-      uses: actions/upload-artifact@v4
-      with:
-        name: python-package-distributions
-        path: dist/
+      - name: Push manifest to ECR
+        working-directory: ./docker
+        run: make ecr-manifest VERSION=${{ needs.set-version.outputs.version }}
+
+      - name: Push manifest to GHCR
+        if: github.event_name == 'push'
+        working-directory: ./docker
+        run: make ghcr-manifest VERSION=${{ needs.set-version.outputs.version }}
+
+  # Currently broken: workflow deploy requires all versions
+  # deploy:
+  #   if: github.event_name == 'push'
+  #   runs-on: ubuntu-latest
+  #   needs: [set-version, push-manifest]
+  #   steps:
+  #     - name: Checkout code
+  #       uses: actions/checkout@v3
+  #       with:
+  #         fetch-depth: 0
+
+  #     - name: Set image
+  #       run: |
+  #         echo "image=471112909428.dkr.ecr.us-east-1.amazonaws.com/arcadeai/arcade-ai:${{ needs.set-version.outputs.version }}" >> $GITHUB_OUTPUT
+
+  #     - name: Deploy to Amazon ECS
+  #       env:
+  #         GITHUB_TOKEN: ${{ secrets.PAT }}
+  #       run: gh workflow -R ArcadeAI/Team run Deploy -f actor-version=${{ needs.set-version.outputs.version }}
```

Comment:
nit: can we remove?

Reply:
I want to reenable it later, but need to fix a different workflow for it to work